package.json

```diff
@@ -39,41 +30,34 @@
 			"name": "Patrick De Young",
 			"email": "hello@patrickdeyoung.me",
 			"url": "https://patrickdeyoung.me"
+		},
+		{
+			"name": "Sukhraj Ghuman",
+			"email": "sukhrajghuman@live.com",
+			"url": "https://github.com/sukhrajghuman"
 		}
 	],
 	"license": "MIT",
 	"devDependencies": {
-		"browser-sync": "^2.18.8",
-		"onchange": "^3.2.1"
+		"autoprefixer": "^9.0.1",
+		"browser-sync": "^2.24.5",
+		"node-sass": "^4.9.2",
+		"onchange": "^4.1.0",
+		"postcss-cli": "^6.0.0"
 	},
 	"dependencies": {
 		"@gov.au/accordion": "latest",
 		"@gov.au/animate": "latest",
-		"@gov.au/body": "latest",
```

Comment:
As a starterkit, it might be wise to add all of the components to get users up and running with the kit and not having to worry about adding more.

This allows them to copy paste html snippets and see them work.